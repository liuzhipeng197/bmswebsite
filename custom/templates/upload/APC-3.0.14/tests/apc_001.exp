string(11) "hello world"
string(11) "hello world"
string(4) "nice"
===DONE===