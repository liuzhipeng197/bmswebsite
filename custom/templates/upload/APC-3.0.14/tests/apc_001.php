<?php

$foo = 'hello world';
var_dump($foo);
apc_store('foo',$foo);
$bar = apc_fetch('foo');
var_dump($bar);
$bar = 'nice';
var_dump($bar);

?>
===DONE===
