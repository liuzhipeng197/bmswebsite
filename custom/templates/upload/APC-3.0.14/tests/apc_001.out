PHP:  Error parsing /opt/bms/custom/templates/upload/APC-3.0.14/tmp-php.ini on line 1231
string(11) "hello world"
string(11) "hello world"
string(4) "nice"
===DONE===